dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -122,7 +122,7 @@ xPoweredBy.name=X-Powered-By
 xPoweredBy.desc=Alternative string for the X-Powered-By header setting. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.xpoweredby. There is no default value for this property. If the property is not set, the value of the X-Powered-By header is set to Servlet/<servlet spec version>, as defined by the Servlet specification.
 
 disableXPoweredBy.name=Disable the X-Powered-By header
-disableXPoweredBy.desc=Disable setting the X-Powered-By header. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.disablexpoweredby. For servlet-5.0 and newer Servlet features, the default value is true.
+disableXPoweredBy.desc=Disable setting the X-Powered-By header. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.disablexpoweredby. For servlet-5.0 and newer Servlet features, the default value is true. Only configurable for Servlet 5.0 or older.
```

Comment:
I don't like how it says and newer servlet features since it's removed from servlet 6.0. 

Maybe reword to say: Only configurable for Servlet 5.0 or older. Default value is false for servlet-4.0 and older, but true for Servlet 5.0.